sr_gui_grasp_controller/src/sr_gui_grasp_controller/grasp_controller.py

```diff
@@ -381,8 +420,15 @@ def __init__(self, context):
         loadUi(ui_file, self._widget)
         context.add_widget(self._widget)
 
+        hand_finder = HandFinder()
+        hand_parameters = hand_finder.get_hand_parameters()
+        hand_serial = hand_parameters.mapping.keys()[0]
```

Comment:
Add a dropdown to select which hand to use see here for an example:

https://github.com/shadow-robot/sr-visualization/blob/F%2378_movement_record/advanced/sr_gui_movement_recorder/src/sr_gui_movement_recorder/movement_recorder.py#L316-L328

https://github.com/shadow-robot/sr-visualization/blob/F%2378_movement_record/advanced/sr_gui_movement_recorder/src/sr_gui_movement_recorder/movement_recorder.py#L387-L392